app/controllers/admin/routes.py

```diff
@@ -147,7 +148,12 @@ def createEvent(templateid, programid=None):
 @admin_bp.route('/eventsList/<eventId>/view', methods=['GET'])
 @admin_bp.route('/eventsList/<eventId>/edit', methods=['GET','POST'])
 def eventDisplay(eventId):
-    # Validate given URL
```

Comment:
Why did you remove this comment and not move it down with the content it was originally with?